test/e2e-kuttl/quota-forest/09-assert.yaml

```diff
@@ -0,0 +1,21 @@
+# Verify AppWrappers finished successfully
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 120
```

Comment:
Nitpick: I think that it would be safer to increase this value to at least 180, or remove it all together.